src/server/model_db.ts

```diff
@@ -70,7 +58,17 @@ export class ModelDB {
 
   constructor(dataDir: string, minUpdates?: number, currentModelId?: string) {
     this.dataDir = dataDir;
+    if (!fs.existsSync(dataDir)) {
+      fs.mkdirSync(dataDir);
+    }
+
     this.modelId = currentModelId || getLatestId(dataDir);
+    if (this.modelId == NO_MODELS_IN_FOLDER) {
+      setupUserModel().then((dict) => {
+        this.writeNewVars(dict.vars as tf.Tensor[]);
+      });
+    }
```

Comment:
When we start the server for the first time, the effect of these lines will be to:
1. create `dist/data/`
2. load the mnist transfer CNN from the internet
3. save its fully connected variable values to `dist/data/<some-timestamp>.json`

However, (3) will actually happen asynchronously (I can't `await` its completion in the constructor), and the server might start before it finishes. This will only be an issue the very first time we start the server, so it might not be worth addressing, but it would probably be better to wait on starting the server until the database is ready to send weights to the client.